admin-default.properties

```diff
@@ -319,3 +318,6 @@ mosip.admin.applicant-details.max.login.count=30
 
 #-------------------Digital Card Api-----------------------------------
 DIGITAL_CARD_STATUS_URL=${mosip.digitalcard.service.url}/v1/digitalcard
+
+# For Spring 3.x we need to specify the ANT Path Matcher for using the existing ANT path patterns.
```

Comment:
this property is not required as we have already there in application-default.properties

Reply:
Done